alibi_detect/cd/_domain_clf.py

```diff
@@ -0,0 +1,103 @@
+from abc import ABC, abstractmethod
+from typing import Callable
+import numpy as np
+from sklearn.svm import SVC
+from sklearn.calibration import CalibratedClassifierCV
+
+
+class _DomainClf(ABC):
```

Comment:
Does it make sense throughout this file to make the naming more conventional, e.g. `x, y, kernel` instead of `c_all, bools, c_kernel`?

Reply:
Fair point, that might be more intuitive for those familiar with the scikit-learn style API